public/i18n/en.yaml

```diff
@@ -966,6 +968,8 @@ modules:
       namespace-description: Namespace where ModuleTemplates are applied. If left empty, the namespace defined in the ModuleTemplates is used. Resources cannot be created in the kyma-system namespace.
       no-module-templates: No ModuleTemplates to add.
       no-source-yaml: No source YAML added.
+      remove-source-yaml: Remove Source YAML
+      remove-source-yaml-desc: Your previously installed modules will remain available, but you won't be able to add any of the following
```

Comment:
What will come next in the sentence? An unordered list? How will it look in the UI? Maybe we should introduce a colon: `following:`?